src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        //int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[new Random().nextInt(Colors.values().length)];
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/49627e89-87b0-4182-85c2-7e78aae36968)

Reply:
1. Removed unnecessary comments
2. Changed the colorSupplier variable from method to class level

Reply:
colorSupplier variable? What is it?
Random should be class level variable